addons/portal/wizard/portal_wizard.py

```diff
@@ -183,7 +180,7 @@ def action_revoke_access(self):
 
         # remove the user from the portal group
         if user_sudo and user_sudo.has_group('base.group_portal'):
-            user_sudo.write({'groups_id': [(3, group_portal.id), (4, group_public.id)], 'active': False})
```

Comment:
I guess we can't just remove `group_portal` because users need to have either `group_user`, `group_portal` or `group_public`?